main.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 // Version holds the current Gitea version
-const Version = "0.9.99.0915"
+var Version = "0.9.99.0915"
```

Comment:
If I remember correctly you can not manipulate `const` from outside? Need to confirm that.